MfciPkg/Docs/Mfci_Integration_Guide.md

```diff
@@ -78,6 +87,9 @@ string into the PCD.
 
 ```
 
+When using `!include MfciPkg/MfciPkg.dsc.inc`, please ensure the platform pcds for
+ `PcdMfciPkcs7RequiredLeafEKU` and `PcdMfciPkcs7RequiredLeafEKU` are included after the `MfciPkg.dsc.inc`.git

```

Comment:
what is this .git at the end?

Reply:
that is a good question. It should not be there.